configuration/scripts/machines/env.narwhal_aocc

```diff
@@ -33,6 +33,8 @@ module load cray-hdf5/1.12.0.4
 setenv NETCDF_PATH ${NETCDF_DIR}
 limit coredumpsize unlimited
 limit stacksize unlimited
+setenv OMP_STACKSIZE 128M
```

Comment:
how do you know what to set OMP_STACKSIZE to? Do you provide guidance on this in the user docs?

Reply:
I did add that information to the documentation (see https://github.com/CICE-Consortium/CICE/pull/680/files#diff-ea684240af50d1d378a32f8805b71db3773d3eb5df8342f1bf262637bc929529).  In fact, that was one of the things that prevented the OpenMP from working on several platforms.  Many platforms had the default set to 0 or 4M which caused seg faults in OpenMP.  When I increased the value, things worked.  There wasn't always a problem in the OpenMP implementation in CICE that caused failures.  Cheyenne has the default set to 64M which is why things were often working there but not elsewhere and gave me a clue.  This took a while to figure out.  I don't know what value is the minimum needed for CICE.  It will depend on the problem size, number of tasks and threads, decomposition, etc.  I'm pretty sure the tests I was running will work with 16M.  So I set the defaults higher than that on some of the platforms where the default was too small.

Reply:
Is it set to at least 16M for all currently supported machines?   Why are some machines as high as 128M, if 16M is enough for the current test suites?

Reply:
I only changed the setting for the machines I have access to and tested.  I'm reluctant to set values for machines I don't have access to.  They may already have reasonable defaults and/or settings that the systems folks prefer you use.  I recognize that I set some machines to 32M, 64M, and 128M.  That was not a careful process.  For other machines, I just left the default value because it worked.  I tried different values on different machines to see what would work and if it worked, I just stuck with it.  I did not try to figure out the minimum value needed for testing or by machine/compiler.  The compilers/implementation may also change in the future, so it makes sense to set it above the minimum required value anyway.  All the values are pretty reasonable as is.   If the OMP_STACKSIZE needed to be set so high that if affected other parts of the memory use, then we'd want to be more careful.  As it is, I think we have a pretty large range that OMP_STACKSIZE could be set without affecting other aspects of the memory/performance (say 16M to 256M) for our testing.  I don't think we need to be particularly precise here.